lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -70,12 +70,29 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		return await this.driver.findElement( textSelector ).sendKeys( text );
 	}
 
-	async insertContactForm() {
+	async insertShortcode( shortcode ) {
 		await this.addBlock( 'Shortcode' );
 		const shortcodeSelector = By.css( 'textarea.editor-plain-text' );
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, shortcodeSelector );
 		const shortcodeTextarea = await this.driver.findElement( shortcodeSelector );
-		return await shortcodeTextarea.sendKeys( '[contact-form][/contact-form]' );
+		return await shortcodeTextarea.sendKeys( shortcode );
+	}
+
+	async insertContactForm( email, subject ) {
```

Comment:
@JavonDavis I propose to extract this as a separate block. We already have some blocks [here](https://github.com/Automattic/wp-e2e-tests/tree/master/lib/gutenberg/blocks). It would be easier for re-use in the future and this file is getting bigger and bigger :)

Reply:
Thanks for the suggestion and I agree it'd be make it easier to reuse this way, I'll make an update for this

Reply:
I've moved contact form, shortcodes and images into separate blocks. Maybe we should get this merged in a little earlier before the contact form is ready?

Reply:
Sounds good! Would you mind to make separate PR for these blocks?